src/algokit/core/bootstrap.py

```diff
@@ -7,29 +8,30 @@
 
 import click
 
+from algokit.core.constants import ALGOKIT_CONFIG
+
 if sys.version_info >= (3, 11):
-    import tomllib
+    pass
```

Comment:
this looks left over

Reply:
Good catch, my vscode auto updated the import and i overlooked it, removed